### ISSUE TITLE
Chai expects are not working. 

### ISSUE THREAD

**Issue by kerimovaa** [mucsi96/nightwatch-cucumber#225]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead use https://groups.google.com/d/forum/nightwatch-cucumber
```

**Current behavior**
The step using "Expect" assertions API (chai js) always not working .

**Expected/desired behavior**
The step using "Expect" is working

**Reproduction of the problem**
`import {client} from 'nightwatch-cucumber';`

```
Then(/^number of orders is more than (\d+)$/, (count) => {
    return client.page.completedOrders()
      .getElementsCount(orderSel, result => {
        if(result)
          client.expect(result).to.be.above(count);
        else
          client.assert.ok(false, 'Something is going wrong');
      });
  });
```

**What is the expected behavior?**
Nightwatch "Expect" assertions API should work 

**What is the motivation / use case for changing the behavior?**
"Expect" API is very easy to use and favored by Nightwatch team.

**Please tell us about your environment:**
* **nightwatch-cucumber version:**
"7.1.2"
* **nightwatch version:**
"0.9.14"
* **cucumber version:**
"2.0.0-rc.9"
* **Node.js version:**
"6.3.1"

**Comment by mucsi96** [comment]
Hi @kerimovaa ! Thanks for feedback! I need a complete runnable example instead of a chunk. Please attach it as ZIP (without including node_modules).

**Comment by kerimovaa** [comment]
I was wrong about operating system, now chai expects are not working everywhere (I've already edited issue). Attached ZIP with feature, step_definition and pages.

**Comment by kerimovaa** [comment]
[test.zip](https://github.com/mucsi96/nightwatch-cucumber/files/1047864/test.zip)

**Comment by mucsi96** [comment]
Thanks! Checking...

**Comment by mucsi96** [comment]
This is probably because of `chai-nightwatch`

**Comment by mucsi96** [comment]
Currently I will do not have time to add support for it because of my university final exam. After 1 month I will check it.

**Comment by mucsi96** [comment]
Hi @kerimovaa ! The issue should be solved from version `8.0.6`. 🎉 

Thank you for your feedback and patient and sorry for so long resolution.

To be able to use the latest version with chai please make sure:
- You are using Cucumber version 3
- You are using `chai` instead of `chai-nightwatch`

I have tested the following step definitions with success:
```
const { client } = require('nightwatch-cucumber');
const { expect } = require('chai');
const { defineSupportCode } = require('cucumber');

defineSupportCode(({ Given, Then, When }) => {
  Given(/^I open Google`s search page$/, () => {
    return client
      .url('http://google.com');
  });

  Then(/^the title is "(.*?)"$/, (text) => {
    return client
      .title(({ value }) => {
        expect(value).to.equal(text);
      });
  });

  Then(/^the Google search form exists$/, () => {
    return client.assert.visible('input[name="q"]');
  });
});
```
